cimetrics/plot.py

```diff
@@ -222,11 +227,16 @@ def split(self, series):
                 ha="right",
             )
 
+    comment = ""
     if not diff_against_self:
-        print(f"Comparing {BRANCH} and {target_branch}")
-        ax.set_title(f"{BRANCH} vs {target_branch}")
+        builds_ids = list(bids)
```

Comment:
could bids ever be an empty list?

Reply:
Yes but we take the other branch then (see :132). This is a very good point though, we don't need a separate diff_against_self variable, I'll fix that.

Reply:
Sorry, no, that's not quite right. We could have some build ids, but no reference metrics, so we need to keep the distinction. It's not possible to have some data but no build ids (because the process is, we look build_ids, then we ask for the data for those).